helper/helper.go

```diff
@@ -363,3 +366,27 @@ func TLScertToFile(filename string, derBytes []byte) error {
 
 	return err
 }
+
+// FormatUploadFilePath ensures that path separators are "/", and returns error if the
+// filepath contains a disallowed character matched with regex
+func FormatUploadFilePath(filePath string) (string, error) {
+
+	// Check for mixed "\" and "/" in filepath. Stop and throw an error if true so that
+	// we do not end up with unintended folder structure when applying ReplaceAll below
+	if strings.Contains(filePath, "\\") && strings.Contains(filePath, "/") {
+		return filePath, fmt.Errorf("filepath contains mixed '\\' and '/' characters")
+	}
```

Comment:
This might be unnecessary in the end, but it will require manual testing to validate since `/` actually is a valid separator on windows.